core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ambari/AmbariLdapService.java

```diff
@@ -1,5 +1,6 @@
 package com.sequenceiq.cloudbreak.service.cluster.ambari;
 
+import static com.sequenceiq.cloudbreak.service.cluster.ambari.AmbariRepositoryVersionService.AMBARI_VERSION_2_7_100_0;
```

Comment:
I think this can be removed, see my comment below

Reply:
It is throwing an UnknownProperty error otherwise..